ugvc/mrd/mrd_utils.py

```diff
@@ -808,22 +787,28 @@ def prepare_data_from_mrd_pipeline(
 
```

Comment:
Please make sure at least one of db_control_signatures_vcf_files, control_signatures_vcf_files, matched_control_signatures_vcf_files is not None
Recommend adding a test that gives just one of these to make sure it doesn't fail